packages/components/select/src/useSelect.ts

```diff
@@ -407,7 +407,10 @@ export const useSelect = (props, states: States, ctx) => {
         originClientHeight ||
         (input.clientHeight > 0 ? input.clientHeight + 2 : 0)
       const _tags = tags.value
-      const gotSize = getComponentSize(selectSize.value || form?.size)
+      const cssVarOfSelectSize =
+        getComputedStyle(input).getPropertyValue('--el-input-height')
```

Comment:
We also need to handle the [namespace](https://element-plus.org/en-US/guide/namespace.html) attribute here.

Reply:
'--el-input-height' -> `ns.cssVarName('input-height')`